themes/default/content/blog/restore-stacks/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Announcing Restore Stacks: Recover Deleted Stacks in the Pulumi Cloud"
+allow_long_title: true
+# The date represents the post's publish date,
+# and by default corresponds with the date this file was generated.
+# Posts with future dates are visible in development,
+# but excluded from production builds.
+# Use the time and timezone-offset portions of of this value
+# to schedule posts for publishing later.
+date: 2023-07-19
+
+# Use the meta_desc property to provide a brief summary
+# (one or two sentences) of the content of the post,
+# which is useful for targeting search results or social-media previews.
+# This field is required or the build will fail the linter test.
+# Max length is 160 characters.
+meta_desc: Pulumi Cloud launches new Restore Stacks feature for Enterprise and Business Critical editions.
+
+# The meta_image appears in social-media previews and on the blog home page.
+# A placeholder image representing the recommended format, dimensions and aspect ratio
+# has been provided for you.
+meta_image: meta.png
+
+# At least one author is required.
+# The values in this list correspond with the `id` properties
+# of the team member files at /data/team/team.
+# Create a file for yourself if you don't already have one.
+authors:
+    - meagan-cojocar
+    - isabel-suchanek
+
+# At least one tag is required.
+# Lowercase, hyphen-delimited is recommended.
+tags:
+    - features
+
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md.
+# for additional details,
+# and please remove these comments before submitting for review.
+---
+
+Starting today, you can restore previously deleted stacks in the Pulumi Cloud console. We've had a number of requests from customers to recover stacks, either because the stack was accidentally deleted or the stack was intentionally deleted but, later on, they want to restore and preserve the activity history on the stack and just remove its resources.
+
+<!--more-->
+
+Customers often use `pulumi destory` to delete the resources in a stack but leave the state file. If they want to delete the state file they use `pulumi stack rm`. Customers might also use `pulumi stack rm --force`, which forces the deletion of the state file but leaves behind its resources and in doing so you lose the ability to manage those resources going forward. There are cases where a force delete is a valid choice, but at Pulumi we sometimes receive customer support tickets when someone runs `pulumi stack rm --force` accidentally. One of the benefits of using Pulumi Cloud is that it maintains state file versions enabling Pulumi to be able to restore deleted stacks. Starting today we have built this recovery process into the product, allowing customers to self-serve restore their stacks from a deleted state.
```

Comment:
Feel free to ignore these:

- should `allowing customers to self-serve restore their stacks ...` be `... to self-serve restoring their stacks ...` or `... to self-serve restoration of their stacks ...`

- and maybe say `... enabling Pulumi to be able to restore deleted state files ...` instead of `... deleted stacks ...` 
Similarly with the last line around `... restore their stacks from a deleted state ...` should be `... restore deleted state files ...` The reason being I worry that someone might think we offer some magic where we'll redeploy resources from a restored state file if we say we are restoring the stack.